### PR TITLE
Checking in updates to camera, zones and regions documentation

### PR DESCRIPTION
**Pull request by nabeelmsft**
Checking in updates to camera, zones and regions documentation

### CODE REVIEW

File: docs/azure_jumpstart_ag/contoso_hypermarket/shopper_insights/_index.md [hunk 1]
```diff
@@ -21,7 +21,68 @@ Store managers can view foot traffic for a store using the Store Manager dashboa
```
[the remark this answers is not in the document]
Comment (nabeelmsft) [reply docs/azure_jumpstart_ag/contoso_hypermarket/shopper_insights/_index.md:21]: This has been resolved by commit:  24e60f49e8a5a65d09549568e4db2cb3f741c87f

**Comment by nabeelmsft** [comment]
> Great work, looks great :)
> 
> * Please use bullets for each step
> * The architecture diagram should be at the start of the page - See other scenarios
> * Please see the Valve errors (spelling, grammar,...etc). Look at the PR checks for that
> * For images requiring multiple steps try to use numbering in circles. See https://raw.githubusercontent.com/Azure/arc_jumpstart_docs/main/docs/azure_jumpstart_arcbox/DataOps/sql_stress_start.png

Thanks @sebassem for the review. Great points. I have addressed all the issues per your suggestions, and they have been checked in. Regarding the architecture diagram, after talking to @dkirby-ms, it was concluded that we will keep the location where it is right now.

File: docs/azure_jumpstart_ag/contoso_hypermarket/shopper_insights/_index.md [hunk 2]
```diff
@@ -11,17 +11,69 @@ Contoso Hypermarket uses computer vision to enhance the customer in-store experi
...
+
+- In order to create a new zone click "+Add Zone" button as shown in the above screenshot. This will start the a set up wizard for zones.
+
+- The first step for the setup wizard for zones is for uploading a floor plan. Clicking the "+Add Zone" will show the page for uploading a floor plan as shown below:
+
+    ![A screenshot showing Upload floor plan step with "Next" button highlighted](./uploadfloorplan.png)
+
+- Clicking the "Next" button in the upload floor plan page, will take the user to "Draw floor zones" page as shown below:
+
+    ![A screenshot draw floor zones page](./drawfloorzones.png)
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/contoso_hypermarket/shopper_insights/_index.md:48]: Please add a ref arrow to show where the user needs to click

**Review by sebassem — CHANGES_REQUESTED** [review]
Minor comments and please refactor the last section to be like [this one ](https://arcjumpstart-ignite.azurewebsites.net/azure_jumpstart_ag/contoso_hypermarket/deployment#next-steps)

File: docs/azure_jumpstart_ag/contoso_hypermarket/shopper_insights/_index.md (hunk 2 above)
Comment (nabeelmsft) [reply docs/azure_jumpstart_ag/contoso_hypermarket/shopper_insights/_index.md:48]: This the image is not meant to show where to click. The purpose of the image is to show how the page will look like when they come to this page after completing the previous step. The subsequent actions are already defined in the next step.

File: docs/azure_jumpstart_ag/contoso_hypermarket/shopper_insights/_index.md [hunk 3]
```diff
@@ -11,17 +11,69 @@ Contoso Hypermarket uses computer vision to enhance the customer in-store experi
...
+
+    This will list all the zones that are available as shown below:
+
+    ![A screenshot showing "Zones" tab highlighted](./zones.png)
+
+- In order to create a new zone click "+Add Zone" button as shown in the above screenshot. This will start the a set up wizard for zones.
+
+- The first step for the setup wizard for zones is for uploading a floor plan. Clicking the "+Add Zone" will show the page for uploading a floor plan as shown below:
+
+    ![A screenshot showing Upload floor plan step with "Next" button highlighted](./uploadfloorplan.png)
```
[the remark this answers is not in the document]
Comment (nabeelmsft) [reply docs/azure_jumpstart_ag/contoso_hypermarket/shopper_insights/_index.md:44]: This has been fixed as part of commit: 4735d4cadd7527856e48bc9a04e108fdc07b8d65

File: docs/azure_jumpstart_ag/contoso_hypermarket/shopper_insights/_index.md (hunk 2 above)
Comment (sebassem) [reply docs/azure_jumpstart_ag/contoso_hypermarket/shopper_insights/_index.md:48]: got you :)

**Pull request merged**